Mathlib/GroupTheory/Submonoid/Membership.lean

```diff
@@ -197,11 +197,11 @@ namespace Submonoid
 @[to_additive]
 theorem mem_iSup_of_directed {ι} [hι : Nonempty ι] {S : ι → Submonoid M} (hS : Directed (· ≤ ·) S)
     {x : M} : (x ∈ ⨆ i, S i) ↔ ∃ i, x ∈ S i := by
-  refine' ⟨_, fun ⟨i, hi⟩ => (SetLike.le_def.1 <| le_iSup S i) hi⟩
+  refine ⟨?_, fun ⟨i, hi⟩ ↦ le_iSup S i hi⟩
```

Comment:
Do you think you might be able to separate out these easy "inline" golfs into a separate PR to reduce the diff?

Reply:
Yeah the golfs to `coe_iSup_of_directed` lemmas are orthogonal to the golfs in FieldTheory/Adjoin. I can easily split them into another PR as they modify other files.

Reply:
I do realize It's unfortuante that the change of the order of the two sections led git diff to align the `Supremum` section but not the `AlgHomMkAdjoinSplits` section, which makes changes to the latter harder to review.

Reply:
Hmm, I'm lucky! I moved Supremum further up in order to golf the compact element proof, and now git diff aligns the AlgHomMkAdjoinSplits section rather than the Supremum section! So it should be much easier to review the main part of the PR (where things get changed rather than moved).